packages/std-client/src/setup/types.ts

```diff
@@ -26,7 +26,7 @@ export type DecodedSystemCall<
   updates: DecodedNetworkComponentUpdate[];
 };
 
-export type ContractComponent = Component<Schema, { contractId: string; tableId?: string }>;
+export type ContractComponent = Component<Schema>;
```

Comment:
does this break any assumptions of existing code? maybe the ecs browser, i believe it's still in use by sky strife? (cc @Kooshaba)